trade-finance-manager-ui/templates/case/underwriting/pricing-and-risk/edit-pricing-and-risk.njk

```diff
@@ -5,6 +5,8 @@
 
 {% extends "index.njk" %}
 
+{% block pageTitle %}Add a credit rating for {{ deal.submissionDetails.supplierName}}{% endblock %}
```

Comment:
Multiline formatting please.

Reply:
Changed all pageTitle blocks to multiline format.